### ISSUE TITLE
app\controller

### ISSUE THREAD

**Issue by stvowi** [aintframework/skeleton_application#2]
Would you please implement dispatching\run_default(actions_namespace, error_handler) in your framework? this function call lead to error

**Comment by tbreuss** [comment]
Same here: 

~~~
[Tue Nov 14 14:49:28 2023] [::1]:49889 [200]: GET / - Uncaught Error: Call to undefined function aint\mvc\dispatching\run_default() in /PhpstormProjects/skeleton_application/src/app/controller.php:26
Stack trace:
#0 /PhpstormProjects/skeleton_application/www/index.php(8): app\controller\run()
#1 {main}
  thrown in /PhpstormProjects/skeleton_application/src/app/controller.php on line 26
~~~

**Comment by tbreuss** [comment]
Fixed with #3
